src/otx/algo/common/losses/iou_loss.py

```diff
@@ -76,6 +78,94 @@ def forward(
         )
 
 
+class IoULoss(nn.Module):
+    """IoULoss.
+
+    Computing the IoU loss between a set of predicted bboxes and target bboxes.
+
+    Args:
+        linear (bool): If True, use linear scale of loss else determined
+            by mode. Default: False.
+        eps (float): Epsilon to avoid log(0).
+        reduction (str): Options are "none", "mean" and "sum".
+        loss_weight (float): Weight of loss.
+        mode (str): Loss scaling mode, including "linear", "square", and "log".
+            Default: 'log'
+    """
+
+    def __init__(
+        self,
+        linear: bool = False,
+        eps: float = 1e-6,
+        reduction: str = "mean",
```

Comment:
```suggestion
        reduction: Literal["none", "mean", "sum"] = "mean",
```
How about using literal as above?

Reply:
Done at 73b7672